docs/plugins/lamp-plugin.md

```diff
@@ -27,6 +27,23 @@ The Probo [Drupal plugin](/plugins/drupal-plugin/), [WordPress plugin](/plugins/
 | `phpConstants` | A hash of constants, such as {const1: 'const1Value', const2: 'const2Value',}. This will overwrite any other auto_prepend_file directives in your php.ini. Accepts a **hash** value. |
 | `phpMods` | An array of php5 modules to enable (should be installed via installPackages if needed). Accepts an **array** value. |
 
+### Modify PHP Environment Options in a Build
+Probo builds have their own isolated `php.ini` files. Specific PHP options for your build can be modified using the `phpIniOptions` configuration option in  your `.probo.yaml` file. The `phpIniOptions` configuration option must be paired with a LAMPApp compatible [Probo Plugin](https://docs.probo.ci/plugins/) to set specific PHP settings within the plugin's steps. 
+
+See below for an example YAML config that sets custom `upload_max_filesize`, `post_max_size`, and `memory_limit` PHP settings values for a Probo Build using the [Probo LAMPApp plugin](/plugins/lamp-plugin/).
```

Comment:
We don't need the last piece of this sentence since they are already on the `/plugins/lamp-plugin` page. Please remove "PHP settings values for a Probo Build using the [Probo LAMPApp plugin](/plugins/lamp-plugin/)."